### DISCUSSION TITLE
How about adding quite logging mode?

### DISCUSSION THREAD

**Discussion by temeddix** [agronholm/apscheduler#595] [Ideas]
I'm using `logging` module from basic libraries of Python, like this:
```
import logging
logging.getLogger().setLevel("INFO")
logging.info("Something to tell")
```
However, I found out that the APScheduler's info logging pollutes the stream ouput, since I'm using `trigger="cron"` and `seconds="*"`.  How about adding quiet logging mode?

**Comment by agronholm** [comment]
The logging module allows for very versatile configuration. You could set the root logger to `WARNING`, and then the loggers you're really interested in to `INFO`. [See for yourself](https://docs.python.org/3/library/logging.config.html).
